hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/Constants.java

```diff
@@ -345,14 +344,14 @@ private Constants() {
       "fs.s3a.connection.request.timeout";
 
   /**
-   * Default duration of a request before it is timed out: {@value}.
+   * Default duration of a request before it is timed out: Zero.
```

Comment:
nit: can we mention that zero means no timeouts

Reply:
its mentioned in the comment on the property name